aspnetcore/fundamentals/websockets.md

```diff
@@ -0,0 +1,98 @@
+---
+title: WebSockets support in ASP.NET Core | Microsoft Docs
+author: tdykstra
+description: What is WebSockets support in ASP.NET Core and how to use it.
+keywords: ASP.NET Core, WebSockets
+ms.author: tdykstra
+manager: wpickett
+ms.date: 03/25/2017
+ms.topic: article
+ms.assetid: 0e0fedcd-a7b4-4479-8ae0-36eab0229d7e
+ms.technology: aspnet
+ms.prod: aspnet-core
+uid: fundamentals/websockets
+---
+
+# Introduction to WebSockets in ASP.NET Core
+
+By [Tom Dykstra](https://github.com/tdykstra) and [Andrew Stanton-Nurse](https://github.com/anurse)
+
+This article explains how to get started with WebSockets in ASP.NET Core. [WebSocket](https://en.wikipedia.org/wiki/WebSocket) is a protocol that enables two-way persistent communication channels over TCP connections. It is used for applications such as chat, stock tickers, games, anywhere you want real-time functionality in a web application.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/websockets/sample). See the [Next Steps](#next-steps) section for more information.
+
+
+## Prerequisites
+
+* ASP.NET Core 1.1 (does not run on 1.0)
+* Any OS that ASP.NET Core runs on:
+  
+  * Windows 7 / Windows Server 2008 and later
+  * Linux
+  * macOS
+
+   Exception: if your app runs on Windows with Kestrel and IIS, or with WebListener, the OS must be Windows 8 / Windows Server 2012 or later
```

Comment:
Let's remove Kestrel from this sentence, I think it might confuse people. It's IIS that has the problem. The fact that Kestrel is the server behind IIS doesn't really matter (though that is the only supported way to use IIS with ASP.NET Core). Running Kestrel on Windows 7 **without** IIS would support WebSockets just fine.

Also, @Tratcher , we're renaming WebListener to HttpSys in 2.0 right? Should we jump on that in this doc to make sure it stays relevant after the rename?

Proposed new text: 

"Exception: if your app runs on Windows with IIS, or the HttpSys server (also known as WebListener), you must use Windows 8 / Windows Server 2012 or later"